api/src/org/labkey/api/reader/jxl/JxlWorkbook.java

```diff
@@ -381,7 +381,7 @@ public CellStyle getCellStyleAt(int i)
     @Override
     public void close()
     {
-        throw new UnsupportedOperationException("method not yet supported");
+        _workbook.close();

```

Comment:
Any chance `_workbook` can be null?
```suggestion
        if (_workbook != null)
            _workbook.close();

```

Reply:
The jxl Workbook factories used to populate that won't return `null`.